scripts/generators/patternfly-3-component/index.js

```diff
@@ -0,0 +1,50 @@
+const { join, resolve } = require('path');
+const { pascalCase } = require('change-case');
+const { ADD_TO_BARREL_FILE } = require('../actionTypes');
+
+const templatesDir = resolve(__dirname, './templates');
+const patternflyReactRoot = resolve(
+  __dirname,
+  '../../../packages/patternfly-react'
+);
+
+function setPF3Generators(plop) {
+  plop.setGenerator('PatternFly 3 Component', {
+    description: 'Component for PatternFly 3 patternfly-react package',
+    prompts: [
+      {
+        type: 'input',
+        name: 'name',
+        message: 'What name should the component have?'
+      }
+    ],
+    actions: ({ name, type }) => {
```

Comment:
type can be removed.  Do we want to support patterns or will it be components only?  If components only typeDir can be removed and hardcoded too.

Reply:
thanks! I will remove the type for now... we have minimal support for Patterns right now so I think we can revisit that if we determine it's needed.

Reply:
updated